src/Servers/HttpSys/src/HttpSysOptions.cs

```diff
@@ -104,15 +108,15 @@ public HttpSysOptions()
         public TimeoutManager Timeouts { get; } = new TimeoutManager();
 
         /// <summary>
-        /// Gets or Sets if response body writes that fail due to client disconnects should throw exceptions or
-        /// complete normally. The default is false.
+        /// Indicate if response body writes that fail due to client disconnects should throw exceptions or complete normally.
```

Comment:
Revert, `Gets or Sets` is standard terminology for properties. Please use it for the remaining properties as well.